src/main/java/net/tylers1066/beaming/Beaming.java

```diff
@@ -27,6 +32,8 @@ public class Beaming extends JavaPlugin implements Listener {
     private static Beaming instance;
     private static Essentials essentials = null;
 
+    private HashMap<Player, Craft> crewCrafts = new HashMap<>();
```

Comment:
I really would like to move away from the design method of putting more maps in the plugin class and move more towards how Movecraft-Combat is done, with a class that houses each feature and all the listeners/functions for that feature.